lib/ttnt/testtask.rb

```diff
@@ -24,9 +30,21 @@ def initialize(rake_test_task)
       define_tasks
     end
 
-    # Task definitions are taken from Rake::TestTask
-    # https://github.com/ruby/rake/blob/e644af3/lib/rake/testtask.rb#L98-L112
+    private
+
+    # Git repository discovered from current directory
+    #
+    # @return [Rugged::Reposiotry]
+    def repo
+      @repo ||= Rugged::Repository.discover('.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.